app/assets/stylesheets/ddr.scss

```diff
@@ -507,6 +507,47 @@ p + #source-collection {
   margin-bottom: 0;
 }
 
+#item-info dd {
+  margin-bottom: 7px;
+}
+
+/* metadata values in lists */
+#item-info ul {
+
+  li { padding-left: 0; margin-bottom: 8px; }
+
+  -moz-column-count: 1;
```

Comment:
You could remove all these browser-prefixed styles to simplify. We're using the autoprefixer gem, which will write them for us somewhere along the pipeline.